src/game_server_handler.py

```diff
@@ -164,8 +168,18 @@ def choose_winner(self, token, winner_token):
             return {'status': constant.STATUS_ERROR,
                     'msg': "Answers not yet available"}
 
-        # TODO: Run code on a VM, currently assumes the code passes
-        self.players[winner_token].add_points(1)
+        results = self.k8s.run_code_and_get_results(
+            self.current_task[0], {winner_token: self.players[winner_token].get_answer()}
+        )
+
+        if results[winner_token]:
+            self.players[winner_token].add_points(1)
+            self.players[token].add_points(1)
+        else:
+            for player in self.players:
+                if player is not winner_token:
```

Comment:
seems that `is` doen't work every time  
```suggestion
                if player != winner_token:
```
https://stackoverflow.com/a/133024